### PR TITLE
V3 Docs Update

### PR DESCRIPTION
**Pull request by AndrewKahr**
#### Changes

- Created V2 Docs Version
- Added V3 Changes as Current Docs

#### Checklist

- [x] Read the contribution [guide](../blob/main/CONTRIBUTING.md) and accept the
      [code](../blob/main/CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: docs/03-github/04-builder.mdx [hunk 1]
```diff
@@ -315,25 +315,7 @@ _**required:** `false`_ _**default:** `androidPackage`_
...
-    androidKeystorePass: ${{ secrets.ANDROID_KEYSTORE_PASS }}
-    androidKeyaliasName: ${{ secrets.ANDROID_KEYALIAS_NAME }}
-    androidKeyaliasPass: ${{ secrets.ANDROID_KEYALIAS_PASS }}
-```
-
-You should also set all the Android Keystore options (see below). Refer to (this
-section)[/docs/github/deployment/android#3-generate-an-upload-key-and-keystore] for keystore setup.
-
-_**required:** `false`_ _**default:** `false`_
+**[Removed in V3] Please use androidExportType instead.**
```
Comment (webbertakken) [inline docs/03-github/04-builder.mdx:318]: Lets not document removed properties.

**Review by webbertakken — COMMENTED** [review]
Your description is missing why we're moving from version 2 to 3.

Do we really need to bump versions already if all we're doing is removing a deprecated parameter?

If the changes are largely in cloud-runner I'd suggest sticking with v2.x until we have actual breaking changes in our public interface.

**Comment by webbertakken** [comment]
@AndrewKahr is this PR still needed, or did you port over all changes to #407?

**Comment by AndrewKahr** [comment]
I ported it over, closing this.

**Pull request closed without merging**